helm/ck-application/files/application.yaml

```
@@ -0,0 +1,56 @@
+bucket:
+  name: contactless-checking
+
+cors:
+  allowed-origins: "{{- $allowedOrigins := .Values.issuer.ipAddress | join ", http://" -}}
+  http://{{ $allowedOrigins }}
+  {{- range $ip := .Values.issuer.ipAddress -}}
+  , http://{{ $ip }}:{{ $.Values.server.faceClient.httpNodePort }}
+  {{- end -}}
+  {{- if .Values.ingress.enabled -}}
+  , http://{{ include "ck-application.ingressPath" (list . (include "ck-client.name" .)) }}
+  {{- end }}"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)